src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const sourceArray = sourceString.split(';');
```

Comment:
The 'split' operation may leave empty strings in the array if there are multiple semicolons without any styles between them. Consider filtering out empty strings to ensure that the resulting object does not contain undefined properties.